CHANGELOG.md

```diff
@@ -2,464 +2,311 @@
+  - Bigint id columns
+  - Memoist gem replaced deprecated ActiveSupport::Memoizable
+- Upgraded AWS SDK gem
+- Upgraded Fog gem
 - LDAP
   - Allow undefined users to log in when “Get User Groups from LDAP” is disabled
   - Ability to set default group for LDAP Authentication
+- Allow Default Zone description to be changed  
+
 
-### [Removed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+16+Ending+Dec+2%2C+2014%22+label%3A%22technical+debt%22)
+### Removed
 
-- SmartProxy (host) directory
+- SmartProxy: 
+  - Removed from UI
+  - Directory removed
+- IP Address Form Field: Removed from UI (use Hostname)
+- Prototype from UI
+- Support for repository refreshes, since they are not used.
+- Support for Host-only refreshes.  Instead, an ESX/ESXi server should be added as a Provider.
 - Rails Fork removal
   - Backport disable_ddl_transaction! from Rails master to our fork
   - Update the main app to use disable_ddl_transaction!
   - Add bigserial support for primary keys to Rails (including table creation and FK column creation)
   - Backport bigserial API to our fork
   - Update application to use new API
-
-### [Fixed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+16+Ending+Dec+2%2C+2014%22+label%3Abug)
-
-- 44 issues fixed. 
-- Notable fixes include:
-  - Fixed issue where deleting a cluster or host tries to delete all policy_events, thus never completing when there are millions of events.
-  - Fixed inheriting tags from resource pool.
-  - Fixed openstack provisioning to deal with multiple security groups with the same name.
-  - Fixed seeding of VmdbDatabase timing out with millions of vmdb_metrics rows
-
-## Unreleased - as of Sprint 15 end 2014-11-10
-
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+15+Ending+Nov+10%2C+2014%22+label%3Aenhancement)
-
-- Security: Lock down [POODLE](http://en.wikipedia.org/wiki/POODLE) attacks.
-- Ruby 2.0
-  - Appliance now built using Ruby 2.0
-  - New commits and pull requests - tested with Ruby 2.0
-- Service Dialogs: Exports can be copied onto an appliance and seeded during appliance startup
-- UI: Continued work on supporting I18N.
-   
-### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+15+Ending+Nov+10%2C+2014%22+label%3Aenhancement)
-
-- Security
-  - Command tool fix_auth now can update passwords in database.yml
-  - Better messaging around overwriting database encryption keys (aka v2_key)
-- More Rails patches removed/upstreamed/backported
-  - Bigint id columns
-  - Memoist gem replaced deprecated ActiveSupport::Memoizable
-- UI: Replaced many legacy Prototype calls with jQuery equivalents.
-- Upgraded AWS SDK gem
-- Upgraded Fog gem
-
-### [Removed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+15+Ending+Nov+10%2C+2014%22+label%3A%22technical+debt%22)
-
 - Old C-Language VixDiskLib binding code
 - Code from product that has been upstreamed into Rails.
```

Comment:
This is really a part of the Rails fork removal bit above.  In addition, we didn't actually remove the fork, so that line up there is misleading.  It should be say something like "reduce need for the Rails fork"